src/main/kotlin/cn/yiiguxing/plugin/translate/MyDynamicBundle.kt

```diff
@@ -0,0 +1,17 @@
+package cn.yiiguxing.plugin.translate
+
+import com.intellij.AbstractBundle
+import com.intellij.DynamicBundle
+import java.util.*
+
+abstract class MyDynamicBundle(bundle: String) : AbstractBundle(bundle) {
+    override fun findBundle(
+        pathToBundle: String,
+        loader: ClassLoader,
+        control: ResourceBundle.Control
+    ): ResourceBundle {
+        val dynamicBundleLocale = DynamicBundle.getLocale()
```

Comment:
It will be `Locale.ENGLISH` if there is no Language Pack installed. If you want to be able to localize Translation plugin separately, you may use a default locale or an explicit setting in this case.